app/utils/purecloud-environment.js

```diff
@@ -1,12 +1,9 @@
-const ENV_REG_EXP = /mypurecloud\.[a-z0-9.]{2,6}$|[a-z0-9]{4,5}.pure.cloud|inin[a-z0-9]{3}\.com|localhost/i;
 import config from '../config/environment';
 
 function purecloudEnvironmentTld() {
-	let env = ENV_REG_EXP.exec(window.location.hostname)[0];
 
-	if (env === 'localhost') {
-		env = 'mypurecloud.com';
-	}
+	let storage = window.localStorage;
```

Comment:
This should use `app/services/storage-service.js`

Reply:
Talked about this. Decide to just use window.localStorage instead of storage-service.